common/djangoapps/entitlements/tests/factories.py

```diff
@@ -13,6 +13,6 @@ class Meta(object):
         model = CourseEntitlement
 
     course_uuid = uuid.uuid4()
-    mode = FuzzyChoice(['verified', 'profesional'])
+    mode = FuzzyChoice(['professional', 'verified', 'no-id-professional', 'credit'])
```

Comment:
Nit: Change this to the Constants as well.

Reply:
I actually had issues when trying to import the constants into this file. I'm not entirely sure why.